### PR TITLE
Bug/id 526/The application crashes after logging in and you try to access the user information section

### LINKED ISSUE #276: Bug/ID-526/The application crashes after logging in and you try to access the UserInformation section
### Severity: Hight

**Linked Test Case Failed:**  https://app.testlodge.com/a/37307/projects/55109/suites/229241?expand_section=404319#case_4016031

**Description:**
After the user logs in with their Google account and attempts to access the section to view or edit their data, the Hermes application unexpectedly closes. However, after a few seconds, the app automatically reopens and allows the user to access the section to view or edit their data. This behavior is inconsistent and interrupts the user's workflow.

**Steps to reproduce:**

1. Open the Hermes application. A screen with a button that says 'Sign Up with Google' will be displayed.
2. Click on the button that says 'Sign Up with Google'. You will be presented with a window with the Google accounts you have stored on your cell phone.
3. Select your Google account. Once selected you will be presented with a window with the map,
4. In the displayed tab click the button with the sandwich icon on the top left. A menu with different options will be displayed.
5. From the menu we will click on the 'User account' option. The app will close immediately.

**Expected result:**
Upon clicking the "User account" option, the app should navigate to the corresponding section, allowing the user to view or edit their data. No unexpected crashes or app closures should occur.

**Current result:**
After selecting the "User account" option, the Hermes application abruptly closes without any indication of an error or warning message.

**Additional Information:**
* Reproducibility: This issue is reproducible consistently by following the steps mentioned above.  
* The issue seems to resolve itself automatically after a few seconds, as the app reopens and allows the user to access the desired section. However, this unexpected closure can be disruptive.

**Attachments:**

Result:

image

Linked Logs:

image

### PR DESCRIPTION
**Pull request by JeferssonCL**

### CODE REVIEW

File: app/src/main/java/com/isc/hermes/SignUpActivityView.java [hunk 1]
```diff
@@ -124,6 +125,14 @@ private void changeActivityToMap(){
         startActivity(intent);
     }
 
+    private void updateInformationUserUsingDB(User user) throws JSONException, ExecutionException,
```
Comment (jhael-arce-jalafund) [inline app/src/main/java/com/isc/hermes/SignUpActivityView.java:136]: Falta javaDocs

File: app/src/main/java/com/isc/hermes/model/User/UserRepositoryUpdaterUsingDBRunnable.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import androidx.annotation.RequiresApi;
+import com.isc.hermes.database.AccountInfoManager;
+import org.json.JSONException;
+import java.util.concurrent.ExecutionException;
+
+public class UserRepositoryUpdaterUsingDBRunnable implements Runnable{
+
+    @RequiresApi(api = Build.VERSION_CODES.O)
+    @Override
+    public void run() {
```
Comment (jhael-arce-jalafund) [inline app/src/main/java/com/isc/hermes/model/User/UserRepositoryUpdaterUsingDBRunnable.java:23]: Falta JavaDocs

File: app/src/main/java/com/isc/hermes/AccountInformation.java [hunk 3]
```diff
@@ -104,6 +104,7 @@ private void updateComponentsByUserInformation() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        System.out.println(UserRepository.getInstance().getUserContained());
```
Comment (CatCoder36) [inline app/src/main/java/com/isc/hermes/AccountInformation.java]: Avoid console printouts

**Review by CatCoder36 — APPROVED** [review]
Good job

File: app/src/main/java/com/isc/hermes/model/User/UserRepositoryUpdaterUsingDBRunnable.java (hunk 2 above)
Comment (JeferssonCL) [reply app/src/main/java/com/isc/hermes/model/User/UserRepositoryUpdaterUsingDBRunnable.java:23]: Done

File: app/src/main/java/com/isc/hermes/SignUpActivityView.java (hunk 1 above)
Comment (JeferssonCL) [reply app/src/main/java/com/isc/hermes/SignUpActivityView.java:136]: Done

File: app/src/main/java/com/isc/hermes/AccountInformation.java (hunk 3 above)
Comment (JeferssonCL) [reply app/src/main/java/com/isc/hermes/AccountInformation.java]: Done

**Review by jhael-arce-jalafund — APPROVED** [review]

**Review by denisgandel — APPROVED** [review]

**Pull request merged**